data/build.gradle

```diff
@@ -33,6 +33,11 @@ android {
 }
 
 dependencies {
+    implementation project(path: ':domain')
+
```

Comment:
too many empty lines

Reply:
Also these libs useless in this module:
androidTestImplementation("io.mockk:mockk-android:$mockk")
    androidTestImplementation "androidx.test.ext:junit:$junit_android"
    androidTestImplementation "androidx.test.espresso:espresso-core:$espresso"
    testImplementation "androidx.test:core:$core"
   
    implementation "androidx.arch.core:core-testing:$arch"
    implementation "io.reactivex.rxjava3:rxandroid:$rxjava"